lib/app.js

```diff
@@ -29,6 +29,24 @@ export default class App extends Component {
 }
 
 class Container extends Component {
+  componentDidMount () {
+    this.scrollToHash()
```

Comment:
We need to do the scrolling here because, in the router there's no way we could know when the correct DOM is available.